blank/HW04.Task1/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Text;
+
+namespace HW04.Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {                
+            char ch;
+            for (int i = 90; i >= 65; i--)
```

Comment:
90, 65 - what a magic numbers? you need to create a constants or local variables for this